etl/steps/data/grapher/ggdc/2022-11-28/penn_world_table.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from owid import catalog
+
+from etl.helpers import Names
+from etl.paths import STEP_DIR
+
+N = Names(__file__)
+VARS_TO_KEEP_FILE = STEP_DIR / "data/grapher/ggdc/2022-11-28/vars_to_keep.csv"
+
+
+def run(dest_dir: str) -> None:
+    dataset = catalog.Dataset.create_empty(dest_dir, N.garden_dataset.metadata)
+
+    table = N.garden_dataset["penn_world_table"]
+
+    # %%
+    # Read in variables to keep file
+    df_vars_to_keep = pd.read_csv(VARS_TO_KEEP_FILE)
```

Comment:
Why don't we send all variables to grapher? It doesn't really matter if have unused variables there.

(In case you really don't want to send them, it might be easier to just add list of variables as a constant to the top of the file rather than reading them from CSV. It's only 32 variables which is totally fine.)

Reply:
There are very specific variables that are useful to have in garden for analysis but not for plotting. In that `vars_to_keep` file there are also the current variable IDs. I will check the `etl-match-variables` function to create the final format that I need.

Reply:
We don't have to plot all variables we have in grapher dataset. It's quite common to have a dataset with 100 variables, but only one is plotted.

Reply:
Actually there are several variables in the dataset currently not used in any chart. It's just we don't want the noise of variables that are useless, like some price levels or several variables which their values are like *Extrapolated (0), benchmark (1) or interpolated (2)*. We can change our mind of course, and that's why they're in `garden`